extensions/functions_rounding.yaml

```diff
@@ -27,3 +27,92 @@ scalar_functions:
           - value: fp64
             name: "x"
         return: fp64
+  -
+    name: "round"
+    description: >
+      Rounding the value `x` to `s` decimal places.
+    impls:
+      - args:
+          - value: fp32
+            name: "x"
+            description: >
+              Numerical expression to be rounded.
+          - value: i32
+            name: "s"
+            description: >
+              Number of decimal places to be rounded to.
+
+              When `s` is a positive number, the rounding
+              is performed to a `s` number of decimal places.
+
+              When `s` is a negative number, the rounding is
+              performed to the left side of the decimal point
+              as specified by `s`.
+        options:
+          rounding:
+            description: >
+              When a boundary is computed to lie somewhere between two values,
+              and this value cannot be exactly represented, this specifies how
+              to round it. For floating point numbers, it specifies the IEEE
+              754 rounding mode (as it does for all other floating point
+              operations). For integer types:
```

Comment:
```suggestion
              to round it:
```

Reply:
These options are a superset of the IEEE 754 options.  Also, I'm not sure what you mean by "For integer types" since this function only applies to floating point types.  I think it's just best if we remove this part altogether.

Reply:
You removed this test below (in the fp64/i32 kernel).  Can you also remove it here (in the fp32/i32 kernel)